mtest/mscore/palette/tst_palette.cpp

```diff
@@ -0,0 +1,104 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2018 Werner Schweer
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#include <map>
+#include <vector>
+#include <string>
+#include <QtTest/QtTest>
```

Comment:
I believe most of these four includes are already done in `all.h`. If so, omit the unnecessary ones here as `all.h` is force-included.